Eatery Blue/UI/EateryCard/EateryExpandableCardDetailView.swift

```diff
@@ -75,7 +64,7 @@ class EateryExpandableCardDetailView: UIView {
         menuCategoryStackView.axis = .vertical
         menuCategoryStackView.alignment = .fill
         menuCategoryStackView.distribution = .equalSpacing
-        menuCategoryStackView.spacing = 8
+        menuCategoryStackView.spacing = 12
```

Comment:
What did kathleen say abt this

Reply:
I asked Liam. We can change this if Kathleen wants.